### PR TITLE
optionally name stories generated from interactive learning

### LINKED ISSUE #3411: optionally name stories generated from interactive learning
We recommend that users name their stories based on what happens, to make them easier to understand, but then output something not useful -- "generated story xxxxxx". We should allow the user to add a name for the story if they want, with the "generated story xxxx" as the default value if they decide not to provide a name.

### PR DESCRIPTION
**Pull request by chidauri**
**Proposed changes**:
- ...Fixes #3411 

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by chidauri** [comment]
@erohmensing does this look good?

**Comment by msamogh** [comment]
Thanks for the PR, we'll give it a review as soon as possible.

**Comment by erohmensing** [comment]
Hi @chidauri, this looks like it will still write each story under the same name (with different numbers) rather than a different name for each story. My original idea would be that whenever you would finish a story and decide to write another, it would ask you for a name, and then that would be stored and used when all the stories were outputted. However, since the output comes from one long tracker that is separated on restart events, I'm not sure if there is actually a clean way to do this. 

TL;DR - this isn't the solution, but I'm not sure there really is a clean solution to this. Happy to discuss further though (maybe on the issue would be more appropriate). WDYT?

**Comment by erohmensing** [comment]
No worries. I think I'll close this issue, too. Hope you continue to contribute after your exams though, thank you for the work you put into this!
